crawlerr/config.py

```
@@ -14,72 +14,79 @@ class CrawlerrSettings(BaseSettings):
+    log_to_file: bool = Field(default=False, alias="LOG_TO_FILE")
     
     # Qdrant Vector Database
-    qdrant_url: str = Field(default="http://localhost:6333", env="QDRANT_URL")
-    qdrant_api_key: Optional[str] = Field(default=None, env="QDRANT_API_KEY")
-    qdrant_collection: str = Field(default="crawlerr_documents", env="QDRANT_COLLECTION")
-    qdrant_vector_size: int = Field(default=1024, env="QDRANT_VECTOR_SIZE")
-    qdrant_distance: str = Field(default="cosine", env="QDRANT_DISTANCE")
-    qdrant_timeout: float = Field(default=10.0, env="QDRANT_TIMEOUT")
-    qdrant_retry_count: int = Field(default=3, env="QDRANT_RETRY_COUNT")
+    qdrant_url: str = Field(default="http://localhost:6333", alias="QDRANT_URL")
+    qdrant_api_key: Optional[str] = Field(default=None, alias="QDRANT_API_KEY")
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)